3-extra/2-sorting-algorithm.js

```diff
@@ -14,7 +14,20 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
-function sortAges(arr) {}
+function sortAges(arr) {
+  const filterArr = arr.filter((item) => typeof item === 'number');
+
+  for (let i = 1; i < filterArr.length; i++) {
+    for (let j = 0; j < i; j++) {
+      if (filterArr[i] < filterArr[j]) {
+        let x = filterArr[i];
+        filterArr[i] = filterArr[j];
+        filterArr[j] = x;
+      }
+    }
+  }
```

Comment:
Good answer! 

An extra thing I would like to point out is, this sorting algorithm is called [bubble sort](https://www.geeksforgeeks.org/bubble-sort/). 

You can also [find out](https://www.geeksforgeeks.org/sorting-algorithms/) there are different types of sorting althorithm which can bring optimization in different scenarios: insertion sort, merge sort, quick sort, heap sort... etc